readthedocs/projects/forms.py

```diff
@@ -85,7 +85,7 @@ class ProjectBasicsForm(ProjectForm):
 
     class Meta:
         model = Project
-        fields = ('name', 'repo', 'repo_type')
+        fields = ('name', 'repo', 'repo_type', 'default_branch')
```

Comment:
Are we showing users a blank `default_branch` field on all manual imports now? This seems..not great, and pretty confusing.

Reply:
For users importing manually yeah. We do show the help text that says that it will be master/trunk if it's empty, not sure if that's enough, we could also make it hidden so users who import from the vcs would get the correct value. But we still want  users who do a manual import to be able to set this value, so not sure.

Reply:
The issue here being that building will be broken if they aren't using a standard VCS-derived default branch?

Reply:
yeah, we will use master, but everyone is using main, or maybe other names and the first build will fail as the repo doesn't have that branch.